coffee/lib/abstract-chosen.coffee

```diff
@@ -162,7 +166,7 @@ class AbstractChosen
           results_group = @results_data[option.group_array_index]
           results += 1 if results_group.active_options is 0 and results_group.search_match
           results_group.active_options += 1
-                
+
```

Comment:
:scissors: Please remove this and all other whitespace-only changes from the PR.